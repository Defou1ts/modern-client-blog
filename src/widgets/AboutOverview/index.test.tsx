import React from 'react';

import { render, screen } from '@testing-library/react';

import renderer from 'react-test-renderer';

import { AboutOverview } from './index';

describe('AboutOverview', () => {
	it('renders a  AboutOverview', () => {
		render(<AboutOverview />);
		const text = screen.getByText('about.overview.title');
		expect(text).toBeInTheDocument();
		expect(text).toBeVisible();
	});

	it('AboutOverview snapshot', () => {
		const component = renderer.create(<AboutOverview />);

		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
