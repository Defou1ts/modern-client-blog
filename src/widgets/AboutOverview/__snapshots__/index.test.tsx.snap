// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutOverview AboutOverview snapshot 1`] = `
<div>
  <div
    className="wrapper"
  >
    <div
      className="about"
    >
      <p
        className="subtitle"
      >
        about.overview.subtitle
      </p>
      <h1
        className="h h1"
      >
        about.overview.title
      </h1>
    </div>
    <div
      className="text"
    >
      <p
        className="p medium"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
      </p>
    </div>
  </div>
  <div
    className="wrapper"
  >
    <div
      className="header"
    >
      <div
        className="statistics"
      >
        <div>
          <h1
            className="h display"
          >
            12+
          </h1>
          <p
            className="p medium"
          >
            about.overview.blogs-published
          </p>
        </div>
        <div>
          <h1
            className="h display"
          >
            18K+
          </h1>
          <p
            className="p medium"
          >
            about.overview.views-on-finsweet
          </p>
        </div>
        <div>
          <h1
            className="h display"
          >
            30K+
          </h1>
          <p
            className="p medium"
          >
            about.overview.total-active-users
          </p>
        </div>
      </div>
      <div
        className="rectangles"
      >
        <div
          className="purpleRectangle"
        />
        <div
          className="yellowRectangle"
        />
      </div>
    </div>
    <div
      className="content"
    />
  </div>
  <div
    className="wrapper"
  >
    <div
      className="item"
    >
      <p
        className="subtitle"
      >
        about.overview.our-mission-subtitle
      </p>
      <h3
        className="h h3 title"
      >
        about.overview.our-mission-title
      </h3>
      <p
        className="p medium text"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Non blandit massa enim nec. Scelerisque viverra mauris in aliquam sem. At risus viverra adipiscing at in tellus.
      </p>
    </div>
    <div
      className="item"
    >
      <p
        className="subtitle"
      >
        about.overview.our-vision-subtitle
      </p>
      <h3
        className="h h3 title"
      >
        about.overview.our-vision-title
      </h3>
      <p
        className="p medium text"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Non blandit massa enim nec. Scelerisque viverra mauris in aliquam sem. At risus viverra adipiscing at in tellus.
      </p>
    </div>
  </div>
</div>
`;
